Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -18,6 +18,7 @@
 #include <fmt/printf.h>
 
 #include "Common/Align.h"
+#include "Common/Assert.h"
```

Comment:
This commit doesn't seem to add any assertions to MemoryViewWidget.cpp. Was this include intended to do something? (I noticed this because it caused a merge conflict on #10957)

Reply:
Must be a leftover from an earlier version of this PR.